central/administration/events/handler/handler.go

```diff
@@ -0,0 +1,92 @@
+package handler
+
+import (
+	"context"
+	"time"
+
+	"github.com/stackrox/rox/central/administration/events/datastore"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/administration/events"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+)
+
+var (
+	_ Handler = (*handlerImpl)(nil)
+
+	flushInterval = time.Minute
```

Comment:
nit: add an env var as a safety net to change this with the default as 1 minute. You never know if you need to tune it and when it's hardcoded, it's a nightmare code fix